fsm-task-manager/src/TaskManager.js

```diff
@@ -0,0 +1,78 @@
+import { doUntil, killProcess } from './utils';
```

Comment:
Method 'start' or 'create' is required for starting/creating new tasks ~= start machine for the object.